test/IntegrationTests/WcfNetFrameworkTests.cs

```diff
@@ -15,15 +15,36 @@
 // </copyright>
 
 #if NET462
+using IntegrationTests.Helpers;
 using Xunit.Abstractions;
 
 namespace IntegrationTests;
 
 public class WcfNetFrameworkTests : WcfTestsBase
 {
+    private const string ServiceName = "Wcf.Client.NetFramework";
+
     public WcfNetFrameworkTests(ITestOutputHelper output)
-        : base("Wcf.Client.NetFramework", output)
+        : base(ServiceName, output)
```

Comment:
ServiceName passed as testAppName is a bit misleading.

Reply:
Addressed by https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/2014/commits/66879c70af6e8cf70d7d89c62196d7e9e6f51aee 

PTAL